File: 1523.py
class Solution:
    def countOdds(self, low: int, high: int) -> int:
        count = 0
        if low % 2: 
            low += 1
            count += 1
        if high % 2:
            high -= 1
            count += 1
        return (high - low) // 2 + count

File: src/1523.cpp
class Solution {
public:
    int countOdds(int low, int high) {
        int count = 0;
        if(low % 2) {
            count++;
            low++;
        }
        if(high % 2) {
            count++;
            high--;
        }
        return (high-low) / 2 + count;
    }
};
